compiler/colormsg.nim

```diff
@@ -0,0 +1,38 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2020 Nim Contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+## This module is for adding colour to error messages from the compiler,
```

Comment:
use color everywhere, not color

(consensus from https://forum.nim-lang.org/t/6991#43944)

Reply:
Yea sorry, my conditioned spelling is hard to lose :D